src/ResourceManager/Advisor/Commands.Advisor/Cmdlets/DisableAzAdvisorRecommendation.cs

```diff
@@ -53,7 +53,7 @@ public class DisableAzAdvisorRecommendation : ResourceAdvisorBaseCmdlet
         /// <summary>
         /// Gets or sets the Resource Id.
         /// </summary>
-        [Parameter(ParameterSetName = IdParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true, HelpMessage = "ResourceID of the recommendation to be suppressed.")]
```

Comment:
This piping parameter set is required - is there a reason you want to remove it?

Reply:
I have fixed it, but got some questions with something. I will take it offline.